extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderHooks.class.php

```diff
@@ -65,7 +65,8 @@ public static function onMakeGlobalVariablesScript( array &$aVars ) {
 
 		if ( $wgEnableVisualEditorExt && \VisualEditorHooks::isAvailable( \RequestContext::getMain()->getSkin() ) ) {
 			$aVars['wgEnablePortableInfoboxBuilderInVE'] = $wgEnablePortableInfoboxBuilderInVE &&
-				\RequestContext::getMain()->getUser()->isAllowed( 'createpage' );
+				// use dummy title in template ns
```

Comment:
Please write an explanation why we do this. Do we care only about [`Title::isNamespaceProtected()`](https://github.com/Wikia/app/blob/dev/includes/Title.php#L2536-L2547) or we want to run all checks it have?